apps/prairielearn/src/pages/instructorCourseAdminStaff/instructorCourseAdminStaff.ts

```diff
@@ -99,7 +99,7 @@ router.post(
       if (uids.length === 0) throw new error.HttpStatusError(400, 'Empty list of UIDs');
 
       // Verify the requested course role is valid
-      if (!['None', 'Previewer', 'Viewer', 'Editor', 'Owner'].includes(req.body.course_role)) {
+      if (!['Editor', 'None', 'Owner', 'Previewer', 'Viewer'].includes(req.body.course_role)) {
```

Comment:
These were ordered in order of access, so changing the order here looks weird. Maybe use `CoursePermissionSchema.shape.course_role.parse()`? Or move that column's enum to a separate schema and use that instead?

Reply:
It's hard to set a proper rule as to what is beneficial or not. I'm fine either way with arrays, as long as cases like these can be properly handled for readability. Relying on the db-types value is probably safer anyway in this context, so switching to that would make it cleaner by default.